wordpress-mobile/WordPress-Android
# Automate updating gutenberg translations

Fixes wordpress-mobile/gutenberg-mobile#939

Gutenberg PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1520

This PR automates adding gutenberg mobile strings to the main `strings.xml` files in order to have those translated by glotpress. It also adds support for plurals in `GutenbergEditorFragment` (need to check that glotpress can parse `<plurals>` tags).

### Testing Instructions
- Checkout the branch locally and make sure the gutenberg-submodule is up to date (or targets `develop`)
- Run `python tools/merge_strings_xml.py`
- Check the main strings.xml at `WordPress/src/main/res/values/strings.xml` and make sure it contains all the strings from gutenberg-mobile/bundle/android
- Build the app to make sure the main xml is valid

PR submission checklist:

- [ ] Update the [release process](https://wp.me/P9ugOq-vy) when the PR lands.
